### PR TITLE
fix(components/kubeflow): PyTorchJob launcher

### PR DESCRIPTION
**Pull request by jagadeeshi2i**
Fixes review comments of PR #5170

### CODE REVIEW

File: components/kubeflow/pytorch-launcher/sample.py [hunk 1]
```diff
@@ -1,36 +1,98 @@
...
+
+    worker_spec_output = namedtuple(
+        "MyWorkerOutput", ["worker_spec"]
+    )
+    return worker_spec_output(worker)
+
+
+worker_spec_op = components.func_to_container_op(
+    create_worker_spec,
+    base_image="tensorflow/tensorflow:1.11.0-py3",
```
Comment (chauhang) [inline components/kubeflow/pytorch-launcher/sample.py:77]: Why do we need tensorflow as the base image here?

**Review by chauhang — CHANGES_REQUESTED** [review]
Please see comments inline

File: components/kubeflow/pytorch-launcher/sample.py [hunk 2]
```diff
@@ -1,36 +1,98 @@
...
+                    }
+                },
+                "spec": {
+                    "containers": [
+                        {
+                            "args": [
+                                "--backend",
+                                "gloo",
+                            ],
+                            "image": "gcr.io/kubeflow-examples/pytorch-dist-mnist:v20180702-a57993c",
```
[the remark this answers is not in the document]
Comment (jagadeeshi2i) [reply components/kubeflow/pytorch-launcher/sample.py]: It has older pytorch. I have created a new image of this.

File: components/kubeflow/pytorch-launcher/sample.py (hunk 1 above)
Comment (jagadeeshi2i) [reply components/kubeflow/pytorch-launcher/sample.py]: I guess simple python image would suffice. I will update it.

**Comment by Bobgy** [comment]
/lgtm

cc @chensun to review

**Comment by Bobgy** [comment]
/approve
I see no reason to wait further, because the updates are addressing last review's feedback.

**Pull request merged**